jobs/build/pre-release/Jenkinsfile

```diff
@@ -0,0 +1,107 @@
+
+
+    // Expose properties for a parameterized build
+    properties(
+        [
+            buildDiscarder(
+                logRotator(
+                    artifactDaysToKeepStr: '',
+                    artifactNumToKeepStr: '',
+                    daysToKeepStr: '',
+                    numToKeepStr: '')),
+            [
+                $class: 'ParametersDefinitionProperty',
+                parameterDefinitions: [
+                    [
+                        name: 'FROM_RELEASE_TAG',
+                        description: 'Build tag to pull from (i.e. 4.1.0-0.nightly-2019-04-22-005054)',
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: ""
+                    ],
+                    [
+                        name: 'NAME',
+                        description: 'Release name (i.e. 4.1.0-rc0)',
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: ""
+                    ],
+                    [
+                        name: 'DESCRIPTION',
+                        description: 'Release description for metadata',
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: ""
+                    ],
+                    [
+                        name: 'PREVIOUS',
+                        description: 'Optional: Tag(s) (comma separated) of release this can upgrade FROM',
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: ""
+                    ],
+                    [
+                        name: 'DRY_RUN',
+                        description: 'Only do dry run test and exit.',
+                        $class: 'BooleanParameterDefinition',
+                        defaultValue: false
+                    ],
+                    [
+                        name: 'MAIL_LIST_FAILURE',
+                        description: 'Failure Mailing List',
+                        $class: 'hudson.model.StringParameterDefinition',
+                        defaultValue: [
+                            'aos-art-automation+failed-release@redhat.com'
+                        ].join(',')
+                    ],
+                    commonlib.mockParam(),
+                ]
+            ],
+            disableConcurrentBuilds()
+        ]
+    )
+
+    commonlib.checkMock()
+
+    buildlib.cleanWorkdir("${env.WORKSPACE}")
+
+    try {
+        sshagent(['aos-cd-test']) {
+            release_info = ""
+            // must be able to access remote registry for verification
+            buildlib.registry_quay_dev_login()
+
+            stage("versions") { prerelease.stageVersions() }
+
+            stage("validation") { prerelease.stageValidation(quay_url) }
+
+            stage("payload") { prerelease.stageGenPayload(quay_url) }
+
+            stage("tag stable") { prerelease.stageTagStable(quay_url) }
```

Comment:
should probably rename this since it's no longer destined for the stable stream... stageTagRelease ?